### PR TITLE
completed html-forms

### PR DESCRIPTION
**Pull request by Blake-Frangoulis**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <p>Email</p>
+      <input type="text" id="comment" name="comment" value="" />
```
Comment (shawnkost) [inline html-forms/index.html:10]: This `input` `type` should be `email` rather than `text`

Comment (shawnkost) [inline html-forms/index.html:10]: There should be a `label` associated with this `input`. Utilize your `p` element on line 9

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <p>Why is coding cool?</p>
+      <select id="simple" name="simple">
+        <option selected>Select an Option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
```
Comment (shawnkost) [inline html-forms/index.html:17]: This `select` should have a `label` associated with it. Utilize your `p` element on line 11

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <label for="HTML">HTML</label>
+      <input type="radio" id="CSS" />
+      <label for="CSS">CSS</label>
+      <input type="radio" id="JavaScript" />
+      <label for="JavaScript">JavaScript</label>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" id="SignUp" />
+      <label for="SignUp">Sign me up!</label>
+      <p>Comments</p>
+      <textarea cols="30" rows="8"></textarea>
```
Comment (shawnkost) [inline html-forms/index.html:29]: This `textarea` should have a `label` associated with it. Utilize your `p` element on line 28

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Blake!

**Comment by Blake-Frangoulis** [comment]
I updated the labels but I am not positive that I did it correctly.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**